libs/application/templates/aosh/work-accident-notification/src/forms/Conclusion.ts

```diff
@@ -0,0 +1,56 @@
+import { buildForm, buildSection } from '@island.is/application/core'
+import { Form, FormModes } from '@island.is/application/types'
+import {
+  information,
+  externalData,
+  sections,
+  overview,
+  employee,
+} from '../lib/messages'
+import { Logo } from '../assets/Logo'
+import { conclusionSection } from './WorkAccidentNotificationForm/ConclusionSection'
+
+export const Conclusion: Form = buildForm({
+  id: 'ConclusionApplicationForm',
+  title: '',
+  logo: Logo,
+  mode: FormModes.COMPLETED,
+  children: [
+    buildSection({
+      id: 'externalData',
+      title: externalData.dataProvider.sectionTitle,
+      children: [],
+    }),
+    buildSection({
+      id: 'announcement',
+      title: externalData.dataProvider.announcement,
+      children: [],
+    }),
+    buildSection({
+      id: 'informationSection',
+      title: information.general.pageTitle,
+      children: [],
+    }),
+    buildSection({
+      id: 'accidentSection',
+      title: sections.draft.accident,
+      children: [],
+    }),
+    buildSection({
+      id: `employeeSection`,
+      title: employee.employee.pageTitle,
+      children: [],
+    }),
+    buildSection({
+      id: `causeAndConsequencesSection`,
+      title: sections.draft.causes,
+      children: [],
+    }),
+    buildSection({
+      id: `overviewSection`,
+      title: overview.general.sectionTitle,
+      children: [],
+    }),
```

Comment:
Is it possible to have the stepper as it instead of faking the steps. This adds strange behaviour like it looks like you can go back to the form from the conclusion and like you can go from the main form and back to the prereq.

If a form only has one step, the title in build section can be left blank and then we don't have no stepper.